flipperdevices/Flipper-Android-App
# Migrate FapHub to new api

**Background**

Migrate FapHub module into new api

**Changes**

- Migrate `target` module
- Migrate `manifest` module
- Migrate `queue` module
- Migrate `button` module

**Test plan**

- Open FapHub
- Install some apps
- See they installed correctly 
- Try delete 
- See it's deleted
- Also try switch on dev branch and do the same
